cc3d/core/SBMLSolverHelper.py

```diff
@@ -111,6 +115,36 @@ def __default_mutable_type(self, obj: Union[object, list, dict], obj_default: Un
         else:
             return obj
 
+    def set_gillespie_integrator_seed(self, seed):
+        if seed > self.get_gillespie_integrator_max_seed():
+            raise ValueError(
+                f"Seed is greater than max allowed value for the seed : {self.get_gillespie_integrator_max_seed()}")
+        CompuCellSetup.persistent_globals.gillespie_integrator_seed = seed
+
+    def get_gillespie_integrator_seed(self) -> Union[int, None]:
+
+        try:
+            return CompuCellSetup.persistent_globals.gillespie_integrator_seed
+        except AttributeError:
+            return None
+
+    def get_gillespie_integrator_max_seed(self) -> int:
+
+        try:
+            return CompuCellSetup.persistent_globals.gillespie_integrator_max_seed
+        except AttributeError:
+            return int(2e9)
+
+    def set_integrator_seed(self, integrator_name: str, rr: RoadRunnerPy):
```

Comment:
Gillespie is the only valid integrator for setting a seed, so is there any reason to require an integrator name as argument?

Reply:
I think so. In case roadrunner team comes up with another algorithm e.g. gillespie_2. it is more of future-proofing - if such thing even exists ...